pbnh/tests/paste_curl_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import pycurl
+import unittest
+import json
+import hashlib
+import tempfile
+import os
+
+from pbnh import conf
+from pbnh import app
+
+config = conf.get_config().get('server')
+host = config.get('bind_ip')
+port = config.get('bind_port')
+URL = 'http://{}:{}/'.format(host, port)
+
+
+class TestPost(unittest.TestCase):
+    def setUp(self):
+        self.c = pycurl.Curl()
+        self.c.setopt(pycurl.URL, URL)
+        self.db_fd, app.app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
Yeah, I know. But if CreateDB is getting the driver from `app.config['DATABASE_DRIVER']` and you set `app.config['DATABASE_DRIVER'] ` in the `setUp()`. Then you get to choose which driver to use, no? The only problem is that you are not creating the database explicitly.

How would you do that?